[화영] Update README.md

rainabak opened the pull request:
erase 구현 내용 정리

## 📌 PR 내용 요약
- SSD Erase 구현 내용 정리 

## 🛠️ 주요 변경 사항
- [ ] 기능 추가
- [ ] 버그 수정
- [ ] 리팩토링
- [ ] 테스트 추가/수정
- [x] 문서 업데이트

변경된 파일이나 함수 중심으로 간단히 정리해줘요:
- README.md : SSD Read  구현 내용 추가

## 🧪 테스트 방법


## ⚠️ 참고 사항

rainabak committed:
Update README.md

 erase 구현 내용 정리

kjo7811 reviewed (approved):
미션 확인했습니다.

kbwcap reviewed (approved):
명세 잘 정리해주셔서 감사합니다. :)

rainabak committed:
Update README.md

SSD - erase 조건에 아래 항목 추가
->  size 가 10보다 큰 경우 ERROR_SIZE 에러 ssd_output.txt 파일 저장하고 size 10 만큼은 erase 진행.

rainabak committed:
Update README.md

rainabak committed:
Update README.md

 내용이 길어서 중간에 Collapse 추가
